lib/woocommerce.js

```diff
@@ -8,65 +8,67 @@ var logger = require('./logger'),
  * @param {object} options:
  * url: the store url with no protocol or trailing forward slash,
```

Comment:
Could you make the parameters for `options` [conform with jsdoc](http://usejsdoc.org/tags-param.html#parameters-with-properties)? E.g.:
`@param {string} options.url - The store url with no protocol or trailing forward slash`